src/Components/LyricsModal.js

```diff
@@ -1,41 +1,14 @@
-import React, { useEffect, useState } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import "./LyricsModal.css";
 import "bootstrap";
 import "popper.js";
 import { getLyrics } from "../axios";
 import logo from "./../assets/Dog-swimming.png";
 import BeatLoader from "react-spinners/BeatLoader";
-import Modal from "react-modal";
+import { default as ReactModal } from "react-modal";
```

Comment:
Why not just do 
` import ReactModal from "react-modal"`

Reply:
yes, changed as per suggestion